app/src/test/java/com/rentminder/HouseholdUnitTest.kt

```diff
@@ -9,8 +9,8 @@ class HouseholdUnitTest {
     @Test
     fun `given a household dto when name is Uptown and members are 5 then name is Uptown and members are five`(){
         var household = Household(1, "Uptown", 5)
-        Assert.assertTrue(household.houseName.equals("Uptown"))
-        Assert.assertTrue(household.houseMembers.equals(5))
+        Assert.assertTrue(household.householdName.equals("Uptown"))
+        Assert.assertTrue(household.householdMembers.equals(5))
```

Comment:
I updated the variable names here to what I changed them to back in the Household DTO.